README.md

```diff
@@ -390,6 +390,47 @@ Default labels will be overridden if there is a name conflict.
 
 `register.clear()` will clear default labels.
 
+### Exemplars
+
+The exemplars defined in the OpenMetrics specification can be enabled on Counter
+and Histogram metric types. The default metrics have support for OpenTelemetry,
+they will populate the exemplars with the labels `{traceId, spanId}` and their
+corresponding values.
+
+The format for `inc()` and `observe()` calls are different if exemplars are
+enabled. They get a single object with the format
+`{labels, value, exemplarLabels}`.
+
+When using exemplars, the registry used for metrics should be set to OpenMetrics
+type (including the global or default registry).
```

Comment:
can we detect if this isn't done and throw a helpful error?

Reply:
https://github.com/siimon/prom-client/pull/482/files#diff-50029c8b3fcc5c34781f42a629d19b95b603a0ab9306ac642e3355e297d0233fR54